src/bin/cargo/commands/publish.rs

```diff
@@ -18,6 +18,7 @@ pub fn cli() -> App {
         ))
         .arg_target_triple("Build for the target triple")
         .arg_target_dir()
+        .arg_package_spec_simple("Package to publish")
```

Comment:
Would it be possible to use `.arg_package()` instead?  That way, it would validate that only one `-p` flag is used by the CLI parser.